### PR TITLE
Update phpdoc-meta.md

### PR DESCRIPTION
**Pull request by Jean85**
This PR fixes the format of the meta documents, and updates the members of the future WG for the reboot of the PSR-5.

Please review the whole text so we can fix the scope of the proposal and push forward the entrance vote.

### CODE REVIEW

File: proposed/phpdoc-meta.md [hunk 1]
```diff
@@ -84,31 +79,28 @@ Cons:
...
 
-* Mike van Riel
+ * Chuck Burgess
 
-### 5.2 Sponsors
+### 5.2 Sponsor
 
-* Phil Sturgeon
-* Donald Gilbert
+ * Mchael Cullum
```
Comment (ashnazg) [inline proposed/phpdoc-meta.md]: M*i*chael

Comment (Jean85) [reply proposed/phpdoc-meta.md]: Fixed, thanks!

**Comment by Jean85** [comment]
@ashnazg please accept the team invitation, so PullApprove will consider you as a valid reviewer for PSR-5.

Do you think that the overall content of the meta doc is ok to ask for the entrance vote? @michaelcullum what do you think?

**Comment by ashnazg** [comment]
@Jean85 I thought I already did accept it... ?

Yes, content looks good to me to start off with.

**Comment by Jean85** [comment]
My bad, it was PullApprove that needed to sync the team.. Now it should be ok!

@michaelcullum, if you agree with this we can merge and call the entrance vote! :tada:

**Comment by ashnazg** [comment]
Bump to Mr. @michaelcullum  :-D

**Pull request merged**